CMakeLists.txt

```diff
@@ -91,7 +91,7 @@ set(enable_cbmc_tests on CACHE BOOL "Whether CBMC tests should be enabled")
 
 set(sat_impl "minisat2" CACHE STRING
     "This setting controls the SAT library which is used. Valid values are
-    'minisat2', 'glucose', 'cadical', 'ipasir-cadical' or 'ipasir-custom'"
```

Comment:
Why was this deleted?

Reply:
I removed `cadical` from the list, since the PR changes the build flow so that `cadical` is always included in the build, so there is no longer a reason to build with `-Dsat_impl=cadical`.